hack/build.sh

```diff
@@ -61,6 +61,10 @@ run() {
     # Format source code
     go_fmt
```

Comment:
should we remove the `go_fmt` now? since, in addition to fixing imports, goimports also formats code in the same style as gofmt.

Reply:
ah, ok. I thought they do different things. Let me check.